Bryn Mawr Classical Review.js

```diff
@@ -159,18 +152,20 @@ var testCases = [
 				"seeAlso": [],
 				"attachments": [
 					{
-						"title": "Review of: The Classical Tradition",
+						"title": "Review of: Hamburg, Museum für Kunst und Gewerbe, Band 2: Unteritalisch rotfigure Keramik. Corpus vasorum antiquorum. Deutschland, Bd. 91; Hamburg Bd. 2"
```

Comment:
John, maybe you did a typo http://bmcr.brynmawr.edu/2013/2013-01-04.html

@aurimasv Normal testing in scaffold is currently broken https://github.com/zotero/scaffold/issues/26 but I have a PR https://github.com/zotero/scaffold/pull/27 for it.